### DISCUSSION TITLE
Responsive Stripe Form?

### DISCUSSION THREAD

**Discussion by timsayshey** [bennypowers/stripe-elements#123] [General]
Hi Benny - Do you know if there's a way to make the stripe form field more compact and responsive for tight interfaces? Thanks!

![IMG_9220](https://github.com/user-attachments/assets/15ba581b-e649-4f13-b2ca-0768e3aa4cef)

**Comment by bennypowers** [comment]
maybe you could use `updateStyle`
https://stripe-elements.netlify.app/#class%3A-stripeelements%2C-stripe-elements
